recipes/zbar/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.10":
+    url: "https://sourceforge.net/projects/zbar/files/zbar/0.10/zbar-0.10.tar.gz"
```

Comment:
There is a GitHub mirror. Please prefer it over SourceForge as GitHub is more reliable 

https://github.com/ZBar/ZBar/archive/0.10.tar.gz

Reply:
I saw it when I started developing the recipe, but it seems like the GitHub repo is missing the `configure` file. It has the `configure.ac` but not `configure`.
The official sources are the ones on `sourceforge` afaik.

Reply:
indeed, Github mirror doesn't provide a configure file, you would need run autoreconf -i, but it requires more dependencies.

Reply:
I think using sourceforge is fine.
@ericriff , if you find yourself in a situation where neither a `CMakeLists.txt` or `configure` script is available,
you can run `autoreconf -i` (as uilianries wrote)
Depending on the `configure.ac` script, this (build) requires `autoconf`, `automake` or `libtool`. All of these are available in CCI.